doc/nest-10.md

```diff
@@ -0,0 +1,14 @@
+# Applying the transactions to your dependency graph
```

Comment:
You miss the one important point about `nest merge`: the full explanations about the two dependency graphs handled by nest:

* The one we are editing with commands like `nest requirement add|remove`
* The one that was saved the last time `nest merge` was called.

When doing `nest merge`, the two are compared to extract a list of transactions to do to go from one to the other, and the old "saved" dependency graph is erased with the current "edit" dependency graph.